OpenSSL/_util.py

```diff
@@ -68,6 +70,23 @@ def native(s):
 
 
 
+def path_string(s):
+    """
+    Convert a Python string to a :py:class:`bytes` string identifying the same
+    path and which can be passed into an OpenSSL API accepting a filename.
+
+    :param s: An instance of :py:class:`bytes` or :py:class:`unicode`.
+
+    :return: An instance of :py:class:`bytes`.
+    """
+    if isinstance(s, binary_type):
+        return s
+    elif isinstance(s, text_type):
+        return s.encode(sys.getfilesystemencoding())
```

Comment:
Are there any issue here with `sys.getfilesystemencoding()` returning `None` in Py2?  The docs say that it can do that.  That's why I suggested `return filename.encode(sys.getfilesystemencoding() or 'ascii')` in #62.

However, it seems like `sys.getfilesystemencoding()` returns `None` if it's the "system default" and `string.encode()` can be passed nothing to use the default.  So maybe it should test for `None` and if it's `None` just call `.encode()` with no arguments.